feature-utils/poly-look/src/react-components/popUps/errorPopup/errorPopup.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import "./errorPopup.css";
+
+const emailLink = `\
```

Comment:
Good as MVP, but I would expect this to be in some configuration file eventually, or asset file.